docs/topics/modules/viewing_build_pipeline_logs.adoc

```diff
@@ -0,0 +1,14 @@
+[id="viewing_build_pipeline_logs"]
```

Comment:
It's very difficult to judge what has changed when we move files, so I wonder if @rkratky's idea about a separate PR for changing file names (if there are any changes within the file) is something to follow in these cases.

Reply:
Sure, sounds like a good convention to follow from henceforth, meanwhile here is an attempt at trying to lessen the pain a bit:

**Existing modules mapped to renamed ones:** 
about_stage_run.adoc > about_pipelines_stage_and_run.adoc
working_with_pipelines.adoc > reviewing_your_staged_application.adoc
approving_build_pipeline.adoc >approving_your_application.adoc
viewing_build_pipeline_oso.adoc > viewing_build_pipeline_logs.adoc
optional_detailed_views.adoc> viewing_application_deployment_details.adoc (Removed the internal Feature flag info as it is Beta, removed otional, as it has some important information relvant to developers and I wonder if we need to add a line or two about scaling pods up and down)
viewing_project_github.adoc > viewing_codebase_github.adoc
viewing_projects_oso>viewing_oso_projects

**Earlier sequence of modules:**
about_stage_run.adoc
working_with_pipelines.adoc 
viewing_build_pipeline_oso.adoc
approving_build_pipeline.adoc 
optional_detailed_views.adoc
viewing_project_github.adoc
viewing_projects_oso

**Current Sequence of modules:**
about_pipelines_stage_and_run.adoc
reviewing_your_staged_application.adoc
approving_your_application.adoc
viewing_build_pipeline_logs.adoc
viewing_application_deployment_details.adoc
viewing_codebase_github.adoc
viewing_oso_projects
Hope this helps.